dymos/transcriptions/common/boundary_constraint_comp.py

```diff
@@ -14,9 +14,12 @@ def initialize(self):
         self._constraints = []
         self._vars = {}
 
-    def setup(self):
+    def configure_io(self):
         """
         Define the independent variables as output variables.
+
+        I/O creation is delayed until configure so that we can determine the shape and units for
```

Comment:
So this resolves Issue #374, mark it as such if that's the case.

Reply:
Looks like it does.